src/d7/gitignore

```diff
@@ -15,3 +15,8 @@ secret.settings.php
 local.settings.php
 
 .aquifer/node_modules
+aquifer.local.json
+
+.DS_Store
```

Comment:
Since these also have nothing to do with aquifer, but are instead products of the user's operating system and file manager, I think they should be left out as well. These kinds of things ought to be added in a [global .gitignore file](http://stackoverflow.com/a/7335487) on each user's system.